src/components/TodoModal/TodoModal.tsx

```diff
@@ -15,30 +23,33 @@ export const TodoModal: React.FC = () => {
               className="modal-card-title has-text-weight-medium"
               data-cy="modal-header"
             >
-              Todo #2
+              Todo #
+              {todo?.id}
```

Comment:
Can you just don't show this part if you don't have a todo here? To avoid this `?`
```
{todo && ( ...